src/Maestro/Client/src/MaestroApiTokenCredential.cs

```diff
@@ -0,0 +1,28 @@
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
```

Comment:
In the previous file, it was "Azure" then "Sytem", in this file, it's the reverse.  We should probably pick one order and stick with it, since these are both new files.

Reply:
Rearranged imports.